File: src/main/java/cn/hirrao/javaee/entity/Blog.java
package cn.hirrao.javaee.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import net.sf.jsqlparser.expression.DateTimeLiteralExpression;

@Getter
@Setter
@AllArgsConstructor
public class Blog {
    @TableId(value = "blogId")
    private Long blogId;
    @TableField("content")
    private String content;
    @TableField("createTime")
    private String createTime;
    @TableField("updateTime")
    private String updateTime;
    private Long uid;
    private String title;
}

File: api/src/main/kotlin/cn/hirrao/javaee/entity/Blog.kt
package cn.hirrao.javaee.entity

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId
import lombok.AllArgsConstructor
import lombok.Getter
import lombok.Setter

@Getter
@Setter
@AllArgsConstructor
class Blog {
    @TableId(value = "blogId")
    private val blogId: Long? = null

    @TableField("content")
    private val content: String? = null

    @TableField("createTime")
    private val createTime: String? = null

    @TableField("updateTime")
    private val updateTime: String? = null
    private val uid: Long? = null
    private val title: String? = null
}
